enforcer/src/main/java/org/wso2/choreo/connect/enforcer/websocket/WebSocketResponseObserver.java

```diff
@@ -91,7 +90,7 @@ public void onNext(WebSocketFrameRequest webSocketFrameRequest) {
                         responseStreamObserver.onNext(webSocketFrameResponse);
                     }
                 } else {
-                    logger.info("Websocket frame type not related to throttling");
+                    logger.debug("Websocket frame type not related to throttling");
```

Comment:
Add more details lke what is the frame type in the message

Reply:
fixed